lib/dpl/provider/azure_webapps.rb

```diff
@@ -0,0 +1,40 @@
+module DPL
+  class Provider
+    class AzureWebApps < Provider
+      def config
+        {
+          "username"     => options[:username] || context.env['AZURE_WA_USERNAME'],
+          "password"     => options[:password] || context.env['AZURE_WA_PASSWORD'],
+          "site"         => options[:site] || context.env['AZURE_WA_SITE']
+        }
+      end
+
+      def git_target
+        "https://#{config['username']}:#{config['password']}@#{config['site']}.scm.azurewebsites.net:443/#{config['site']}.git"
```

Comment:
Hm, good point - I guess we can send all the output to dev/null. We'll lose in-line information coming back about the deployment, but that stuff is also available on Azure. I'll update the PR.